### ISSUE TITLE
Reconstruction quality of neus and neus-facto on DTU dataset

### ISSUE THREAD

**Issue by wujinhu1999** [autonomousvision/sdfstudio#266]
Hello author, I did several experiments and found that the reconstruction quality of neus on the DTU dataset is higher than neus-facto. Is this correct?The chamfer distance evaluation index is generally that NeuS is lower than neus-facto.Looking forward to your reply.

**Comment by niujinshuchong** [comment]
Yes, I think so. NeuS-facto is using hash encoding by default which is easier to produce noise results. It is also trained for less iterations.
